Readme.md

```diff
@@ -62,6 +62,8 @@ pino app.js | pino-pretty
   error like objects. Default: `err,error`.
 - `--messageKey` (`-m`): Define the key that contains the main log message.
   Default: `msg`.
+- `--changeLevelName` (`--changeLevelName`): Define the key that contains the level of the log.
+  Default: `level`.
```

Comment:
@mcollina I suggested this be changed to levelKey, the reason it's called `changeLevelName` is because we have that option in the programmatic API.  https://github.com/pinojs/pino/blob/master/docs/api.md#changelevelname-string

I think that's a terrible name in the API and we should deprecate it, alias to `levelKey`, and then remove the alias in v6

If we do this, then this PR should update to `levelKey`

Reply:
see https://github.com/pinojs/pino-pretty/pull/92#discussion_r356183447